docs/workflow/testing/coreclr/unix-test-instructions.md

```diff
@@ -74,16 +74,19 @@ Is used to disable the build, that way if building on Unix cycles are saved buil
 PAL tests
 ---------
 
-Build CoreCLR on the Unix machine.
+Build CoreCLR with PAL tests on the Unix machine:
 
-Run tests:
-
-```
-~/runtime$ src/coreclr/src/pal/tests/palsuite/runpaltests.sh ~/runtime/artifacts/obj/coreclr/Linux.x64.Debug
+```sh
+cd runtime
```

Comment:
This `cd runtime` seems confusing, it makes me think as if I should cd into a subfolder named "runtime" in the runtime repo. I know what you meant, but all the examples above just assume that we are in the root of the runtime repo, which is reasonable.

Reply:
Updated few others as well, so all commands look alike.